src/main/java/com/iridium/iridiumskyblock/DataConverter.java

```diff
@@ -1,66 +1,87 @@
                 .forEach(file -> {
                     try {
                         Path source = Paths.get(file.getAbsolutePath());
                         Path target = Paths.get(file.getParentFile().getAbsolutePath() + File.separator + "old_data" + File.separator + file.getName());
-                        iridiumSkyblock.getLogger().info("Moving " + file.getName() + " to " + target.toFile().getAbsolutePath() + "...");
+                        IridiumSkyblock.getInstance().getLogger().info("Moving " + file.getName() + " to " + target.toFile().getAbsolutePath() + "...");
                         Files.move(source, target);
                     } catch (IOException e) {
                         e.printStackTrace();
                     }
                 });
 
-        Arrays.stream(Objects.requireNonNull(new File(iridiumSkyblock.getDataFolder().getAbsolutePath() + File.separator + "schematics" + File.separator)
+        Arrays.stream(Objects.requireNonNull(new File(IridiumSkyblock.getInstance().getDataFolder().getAbsolutePath() + File.separator + "schematics" + File.separator)
                 .listFiles((file, fileName) -> fileName.endsWith(".schem") || fileName.endsWith(".schematic"))))
                 .forEach(file -> {
                     try {
                         Path source = Paths.get(file.getAbsolutePath());
                         Path target = Paths.get(file.getParentFile().getParentFile().getAbsolutePath() + File.separator + "old_data" + File.separator + "schematics" + File.separator + file.getName());
-                        iridiumSkyblock.getLogger().info("Moving " + file.getName() + " to " + target.toFile().getAbsolutePath() + "...");
+                        IridiumSkyblock.getInstance().getLogger().info("Moving " + file.getName() + " to " + target.toFile().getAbsolutePath() + "...");
                         Files.move(source, target);
                     } catch (IOException e) {
                         e.printStackTrace();
                     }
                 });
     }
 
+    public static void deleteDuplicateRecords() {
+        IridiumSkyblock.getInstance().getDatabaseManager().getIslandBankTableManager().deleteDuplicates();
+        IridiumSkyblock.getInstance().getDatabaseManager().getIslandBlocksTableManager().deleteDuplicates();
+        IridiumSkyblock.getInstance().getDatabaseManager().getIslandBoosterTableManager().deleteDuplicates();
+        IridiumSkyblock.getInstance().getDatabaseManager().getIslandMissionTableManager().deleteDuplicates();
+        IridiumSkyblock.getInstance().getDatabaseManager().getIslandSpawnersTableManager().deleteDuplicates();
+        IridiumSkyblock.getInstance().getDatabaseManager().getIslandUpgradeTableManager().deleteDuplicates();
+    }
+
+    public static void updateDatabaseData(int oldVersion, int newVersion, ConnectionSource connectionSource) {
+        System.out.println("Updating database from version " + oldVersion + " to " + newVersion);
```

Comment:
Any reason why we're not using getLogger?

Reply:
its same -_-